### ISSUE TITLE
formating

### ISSUE THREAD

**Issue by neuri48** [DIGITALCRIMINALS/OnlyFans#479]
1. i think it would be better to remove the "free" folder from the filepath formating, my current setting is "site_name/model_username/media_type//value/ and its good but the "free" folder is kind of an inconvenience

**Comment by Beareschal** [comment]
Remove the {value} field

{value} = Value of the content. Paid or Free.

**Comment by neuri48** [comment]
> Remove the {value} field
> 
> {value} = Value of the content. Paid or Free.

the point is for the paid content to be seperated from everything else. the "free" subfolder is kind of unnecessary since, if it isn't "paid" then its obviously free

**Comment by Beareschal** [comment]
> > Remove the {value} field
> > {value} = Value of the content. Paid or Free.
> 
> the point is for the paid content to be seperated from everything else. the "free" subfolder is kind of unnecessary since, if it isn't "paid" then its obviously free

So you're thinking something like:
site_name/model_username/media_type/xxx/

In this scenario, the folder xxx would contain a dump of all free media posted and a single folder that just contains paid content? Something like that?

**Comment by neuri48** [comment]
> 

exactly, a folder for everything that is posted and then a sub folder for everything that is exclusive to you (paid)

**Comment by Beareschal** [comment]
I like the idea, especially if you participate when you think about communities out there that provide "Leaks". 99& of those leaks simply come from free trials and such. I'm always searching for somewhere that I'd be mutually benefitted by sharing my paid content.

I've never actually looked through his code much but If I ever have some free time...

**Comment by Beareschal** [comment]
In the meantime however:
"{site_name}/{model_username}/{media_type}/{value}"
You should be able to use a tool or make a script to perform what you're wanting, based on that file_directory_format. Could potentially mess with metadata though...
